urdf/package.xml

```diff
@@ -24,12 +24,12 @@
   <build_depend>console_bridge</build_depend>
   <build_depend>urdfdom</build_depend>
   <build_depend>urdfdom_headers</build_depend>
-  <build_depend>TinyXML</build_depend>
+  <build_depend>tinyxml</build_depend>
   <build_depend>tinyxml_vendor</build_depend>
 
   <exec_depend>urdfdom</exec_depend>
-  <exec_depend>urdfdom-headers</exec_depend>
-  <exec_depend>TinyXML</exec_depend>
+  <exec_depend>urdfdom_headers</exec_depend>
```

Comment:
We recently updated the REP to allow ROS packages to have names like `urdfdom-headers`. That makes it unnecessary to change the rosdep keys in downstream packages to e.g. `urdfdom_headers`. We should make use of that and not require these changes to change.